### PR TITLE
fix: forking the workshopper print command so we can have wider screens

### LINKED ISSUE #9: Markdown wrapping a bit eager
The wrapping on content is a bit too eager and causes breaks in a lot of things that are either being highlighted as code or are there to be copy-pasted (and the line break breaks this).

Screen Shot 2021-07-23 at 3 20 43 PM

### PR DESCRIPTION
**Pull request by erunion**
This forks the internal `print` command that `workshopper-adventure` offers so we can have wider Markdown screens and avoid text that's intended to be copy-pasted from being broken up into multiple lines. We need to fork the command unfortunately because `workshopper-adventure` doesn't expose or allow overrides for the options it supplies to [msee](https://npm.im/msee) for Markdown rendering.

Fixes #9 

### Before
Screen Shot 2021-07-23 at 4 52 32 PM

### After
Screen Shot 2021-07-23 at 4 52 25 PM

### CODE REVIEW

File: lib/mseePipe.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  const buffer = [];
+  return through(
+    function (contents, encoding, done) {
+      buffer.push(contents.toString());
+      done();
+    },
+    function (done) {
+      const contents = buffer.join('\n');
+
+      let str = msee(contents).replace(/^/gm, ' ').replace(/$/gm, '  ');
```
Comment (dok) [inline lib/mseePipe.js:21]: Hmm approaching slightly dangerous territory manipulating markdown 😬

File: lib/msee.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    right: '   ',
+    first: '  » ',
+    regular: '    ',
+  },
+  defaultCodePad: '    ',
+  paragraphPad: {
+    left: ' ',
+    right: '   ',
+  },
+  maxWidth: 98, // 100 - 2 for the 2 character indent that bulletpoints add.
```
Comment (dok) [inline lib/msee.js]: Is it possible to take the full width of the screen?

Comment (erunion) [reply lib/msee.js]: Looking through the `msee` code `-1` might do that?

https://github.com/firede/msee/blob/f2d3699076e058245b3c4acd9e40f2a69a58963c/lib/msee.js#L358-L360

Comment (dok) [reply lib/msee.js]: If so, it would be awesome, since folks usually like to control width by adjusting their terminal itself

Comment (erunion) [reply lib/msee.js]: Screen Shot 2021-08-13 at 10 26 18 AM

Yep -1 does it. Pushed that fix into this PR.

**Review by dok — APPROVED** [review]

**Pull request merged**